internal/tjson/schema_test.go

```diff
@@ -67,3 +67,152 @@ func TestSchemaMarshalUnmarshal(t *testing.T) {
 
 	assert.Equal(t, expected, actual)
 }
+
+func TestSchemaEqual(t *testing.T) {
+	t.Parallel()
+
+	cInt64Schema := Schema{
```

Comment:
What is this `c` prefix mean?

Reply:
It means "case" (like "test case").  I didn't find a better word (but suggestions are welcome!)
The idea is to call it differently than just `int64Schema` because such a name already exists in the package.

Reply:
I spent a few minutes, but nothing comes to mind. Maybe we can replace that prefix with the word `case` to make it more clear?

Reply:
Done!